src/app/app.component.ts

```diff
@@ -6,5 +6,5 @@ import { Component } from '@angular/core';
   styleUrls: ['./app.component.scss']
 })
 export class AppComponent {
-  title = 'codemortals';
+  title = 'Codemortals';
```

Comment:
Please can you change this to say `Code Mortals`

Reply:
cool. Will do it

Reply:
Done !!